src/main/java/com/alkemy/ong/controller/CategoryController.java

```diff
@@ -0,0 +1,24 @@
+package com.alkemy.ong.controller;
+
+import com.alkemy.ong.service.CategoryService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/")
+public class CategoryController {
+
+    @Autowired
+    private CategoryService categoryService;
+
+    @GetMapping(value = "/categories", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity getCategories(){
```

Comment:
esto tira sonar "Raw types should not be used" ponerle tipo al ResponseEntity

Reply:
se le puso el tipo de dato que recibe